src/main/java/dev/j3fftw/litexpansion/catergory/MainCategory.java

```diff
@@ -0,0 +1,98 @@
+package dev.j3fftw.litexpansion.catergory;
+
+import dev.j3fftw.litexpansion.Items;
+import dev.j3fftw.litexpansion.LiteXpansion;
+import io.github.thebusybiscuit.slimefun4.api.player.PlayerProfile;
+import io.github.thebusybiscuit.slimefun4.core.categories.FlexCategory;
+import io.github.thebusybiscuit.slimefun4.core.guide.SlimefunGuideLayout;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import io.github.thebusybiscuit.slimefun4.utils.ChestMenuUtils;
+import me.mrCookieSlime.CSCoreLibPlugin.general.Inventory.ChestMenu;
+import me.mrCookieSlime.Slimefun.cscorelib2.item.CustomItem;
+import org.bukkit.ChatColor;
+import org.bukkit.NamespacedKey;
+import org.bukkit.Sound;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+
+public class MainCategory extends FlexCategory {
+
+    private ChestMenu menu;
+
+    public static final MainCategory INSTANCE = new MainCategory();
+
+    public MainCategory() {
+        super(new NamespacedKey(LiteXpansion.getInstance(), "test_category"),
+            new CustomItem(Items.GENERATOR, "&5test")
+        );
+
+        setupInv();
+        menu.setEmptySlotsClickable(false);
+    }
+
+    private void setupInv() {
```

Comment:
Also move this to Open, or add a ChestMenu field to it and just move the new ChestMenu